cmd/podman/varlink/io.podman.varlink

```diff
@@ -435,6 +435,24 @@ type Runlabel(
     opts: [string]string
 )
 
+# EventInput describes the input to obtain libpod events
+type EventInput(
+    filter: []string,
```

Comment:
If you use an `enum` for `Event.id` (see below), you should use that here as well.